packages/hooks/README.md

```diff
@@ -16,19 +16,27 @@ Composable hooks for networking data
 
 ### useCancelling()
 
+[Aborts](https://developer.mozilla.org/en-US/docs/Web/API/AbortController) inflight request is the parameters
+change.
```

Comment:
nitL: hard wrapping seems weird and increases diff churn
```suggestion
[Aborts](https://developer.mozilla.org/en-US/docs/Web/API/AbortController) inflight request is the parameters change.
```

Reply:
What is hard wrapping?

Reply:
adding a newline in the middle of a sentence/paragraph - usually to ensure line lengths are under an arbitrary threshold. Instead, it's best if a sentence/paragraph is as long as it is, allowing the display side to do the line breaking.

Reply:
Oh I should probably change my editor settings for .md files, I usually randomly put endlines cuz it doesn't wordwrap.